tests/coalaHTMLTest.py

```diff
@@ -40,10 +54,14 @@ def test_output_file(self):
                           "-c", os.devnull,
```

Comment:
It feels wrong to have running coala_html.main two times, but testing stuff once. Probably, this should be two separate test cases.